data/documents-schema.json

```diff
@@ -0,0 +1,81 @@
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$ref": "#/definitions/KlimawatchDocuments",
+  "definitions": {
+    "KlimawatchDocuments": {
+      "type": "object",
+      "properties": {
+        "data": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "entity": {
+                "$ref": "#/definitions/Entity"
+              },
+              "documents": {
+                "type": "array",
+                "items": {
+                  "$ref": "#/definitions/Document"
+                }
+              }
+            },
+            "additionalProperties": false,
+            "required": ["entity", "documents"]
+          }
+        }
+      },
+      "required": ["data"]
+    },
+    "Entity": {
+      "title": "Gebietskörperschaft: Landkreis, Gemeinde, etc",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "name": {
+          "type": "string"
+        },
+        "adminLevel": {
+          "type": "integer"
+        },
+        "ags": {
```

Comment:
Was bezeichnet das Property `ags`?

Reply:
Den Amtlichen Gemeindeschlüssel, also das Wikidata Property https://www.wikidata.org/wiki/Property:P439 – wir sollten vielleicht auch noch den ARS (Amtlicher Regionalschlüssel) mit aufnehmen, vgl. https://openknowledgegermany.slack.com/archives/CQC9RQ1NU/p1635673845027400?thread_ts=1635672345.022800&cid=CQC9RQ1NU allerdings fehlt der aktuell noch in Wikidata...

Reply:
Okay, ich ging zu dem Zeitpunkt noch davon aus das AGS und ARS enger verwandt sind... können wir dann auch einfach rauslassen und statt dessen nur den ARS aufführen.